lib/rpc_defs.py

```diff
@@ -304,6 +304,11 @@ def _TestDelayTimeout(duration):
     ("extra", None, "Extra info for device (dev_path for disk)"),
     ("seq", None, "Device seq"),
     ], None, None, "Hoplug a device to a running instance"),
+  ("resize_disk", SINGLE, None, constants.RPC_TMO_NORMAL, [
+    ("instance", ED_INST_DICT, "Instance object"),
+    ("disk", None, "The dist to be resized"),
+    ("new_size", None, "The new disk size in bytes"),
+    ], None, None, "Notify the HV about a disk resize"),
```

Comment:
For growing a disk, there is already `blockdev_grow` RPC. Wouldn't it make sense to also QMP-notify the hypervisor directly from there, without a new RPC?

Reply:
Just for the record: it has been discussed somewhere else, that the `blockdev_grow` RPC is called several times (dry-run, primary and in case of DRBD secondary node). Technically it deals with storage only an not hypervisor stuff. So the extra RPC seems fine here.